### PR TITLE
SlowFast 번역

### PR DESCRIPTION
**Pull request by cpprhtn**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #45 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
SlowFast 모델에 대한 번역을 진행하였습니다.

### CODE REVIEW

File: facebookresearch_pytorchvideo_slowfast.md [hunk 1]
```diff
@@ -48,18 +48,19 @@ from pytorchvideo.transforms import (
...
 ```python 
-# Set to GPU or CPU
+# GPU 또는 CPU 방식을 설정합니다.
 device = "cpu"
 model = model.eval()
 model = model.to(device)
 ```
 
 Download the id to label mapping for the Kinetics 400 dataset on which the torch hub models were trained. This will be used to get the category label names from the predicted class ids.
+토치 허브 모델이 훈련된 Kinetics 400 데이터셋에 대해 레이블 매핑이 된 ID를 다운로드합니다. 이는 예측된 클래스 ID에서 카테고리 레이블 이름을 가져오는데 사용됩니다.
```
Comment (zhoonit) [inline facebookresearch_pytorchvideo_slowfast.md:63]: 목적어가 'the id to label mapping' 이기 때문에
'레이블 매핑이 된 ID 다운로드' -> 'id에서의 label 매핑'을 다운로드  정도로 번역하는 것은 어떨까요??

Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md:63]: 헉.. 감사합니다. [3D ResNet](https://github.com/PyTorchKorea/hub-kr/pull/63)에도 동일한 형태로 번역해두었는데 둘다 반영하겠습니다!

Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md:63]: `ID에서의 레이블 매핑 정보를 다운로드합니다` 의 문장으로 수정했습니다!

File: facebookresearch_pytorchvideo_slowfast.md [hunk 2]
```diff
@@ -143,54 +144,53 @@ try: urllib.URLopener().retrieve(url_link, video_path)
...
-# Initialize an EncodedVideo helper class and load the video
+# EncodedVideo helper 클래스를 초기화하고 비디오를 불러옵니다.
 video = EncodedVideo.from_path(video_path)
 
-# Load the desired clip
+# 원하는 클립을 불러옵니다.
 video_data = video.get_clip(start_sec=start_sec, end_sec=end_sec)
 
-# Apply a transform to normalize the video input
+# 비디오 입력을 정규화하기 위해 transform 함수를 적용합니다.
```
[the remark this answers is not in the document]
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md]: 헛 감사합니다. 반영하겠습니다 :)

**Review by Taeyoung96 — COMMENTED** [review]
@cpprhtn 
번역 하시느라 고생하셨습니다. :smile:

File: facebookresearch_pytorchvideo_slowfast.md [hunk 3]
```diff
@@ -48,18 +48,19 @@ from pytorchvideo.transforms import (
...
 
 ```python 
-# Set to GPU or CPU
+# GPU 또는 CPU 방식을 설정합니다.
 device = "cpu"
 model = model.eval()
 model = model.to(device)
 ```
 
 Download the id to label mapping for the Kinetics 400 dataset on which the torch hub models were trained. This will be used to get the category label names from the predicted class ids.
```
[the remark this answers is not in the document]
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md:62]: 제가 영어 원문보면서 번역한뒤에 지우는데,, 저문장 지우는걸 까먹은거 같습니다!!
지우도록 하겠습니다 :)

File: facebookresearch_pytorchvideo_slowfast.md [hunk 4]
```diff
@@ -143,54 +144,53 @@ try: urllib.URLopener().retrieve(url_link, video_path)
 except: urllib.request.urlretrieve(url_link, video_path)
 ```
 
-Load the video and transform it to the input format required by the model.
+비디오를 불러오고 모델에 필요한 입력 형식으로 변환합니다.
 
 ```python
-# Select the duration of the clip to load by specifying the start and end duration
-# The start_sec should correspond to where the action occurs in the video
+# 시작 및 종료 기간을 지정하여 불러올 클립의 기간을 선택합니다.
```
[the remark this answers is not in the document]
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md:150]: 감사합니다 👍

File: facebookresearch_pytorchvideo_slowfast.md [hunk 5]
```diff
@@ -143,54 +144,53 @@ try: urllib.URLopener().retrieve(url_link, video_path)
...
 inputs = video_data["video"]
 inputs = [i.to(device)[None, ...] for i in inputs]
 ```
 
-#### Get Predictions
+#### 예측값 구하기
 
 ```python
-# Pass the input clip through the model
+# 모델을 통해 입력클립을 전달합니다.
```
[the remark this answers is not in the document]
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md:172]: 좋은의견 감사합니다

File: facebookresearch_pytorchvideo_slowfast.md [hunk 6]
```diff
@@ -16,19 +16,19 @@ accelerator: “cuda-optional”
 demo-model-link: https://huggingface.co/spaces/pytorch/SlowFast
 ---
 
-### Example Usage
+### 사용 예시
 
 #### Imports
```
Comment (codingbowoo) [inline facebookresearch_pytorchvideo_slowfast.md:21]: 해당 항목도 번역 부탁드립니다. 아래 번역하신 내용 상 "불러오기" 정도면 괜찮을 것 같다고 생각합니다 :)

File: facebookresearch_pytorchvideo_slowfast.md [hunk 7]
```diff
@@ -48,18 +48,18 @@ from pytorchvideo.transforms import (
...
 ```python 
-# Set to GPU or CPU
+# GPU 또는 CPU 방식을 설정합니다.
 device = "cpu"
 model = model.eval()
 model = model.to(device)
 ```
 
-Download the id to label mapping for the Kinetics 400 dataset on which the torch hub models were trained. This will be used to get the category label names from the predicted class ids.
+토치 허브 모델이 훈련된 Kinetics 400 데이터셋에 대해 ID에서의 레이블 매핑 정보를 다운로드합니다. 이는 예측된 클래스 ID에서 카테고리 레이블 이름을 가져오는데 사용됩니다.
```
Comment (codingbowoo) [inline facebookresearch_pytorchvideo_slowfast.md:62]:
```suggestion
토치 허브 모델이 훈련된 Kinetics 400 데이터셋을 위한 id-레이블 매핑 정보를 다운로드합니다. 이는 예측된 클래스 id에 카테고리 레이블 이름을 붙이는 데 사용됩니다.
```
의미상 일부 번역을 수정할 것을 제안드립니다. 
두번째 문장에는 의역을 과하게(?) 넣어 봤는데요, 입맛에 맞게 수정해주시면 좋을 것 같습니다 :)

File: facebookresearch_pytorchvideo_slowfast.md [hunk 8]
```diff
@@ -72,13 +72,13 @@ except: urllib.request.urlretrieve(json_url, json_filename)
 with open(json_filename, "r") as f:
     kinetics_classnames = json.load(f)
 
-# Create an id to label name mapping
+# 레이블 이름 매핑에 대한 ID 만들기
```
Comment (codingbowoo) [inline facebookresearch_pytorchvideo_slowfast.md:75]:
```suggestion
# id-레이블 이름 매핑 만들기
```
위와 동일하게 의미상 일부 번역 수정을 제안드립니다.

File: facebookresearch_pytorchvideo_slowfast.md [hunk 9]
```diff
@@ -72,13 +72,13 @@ except: urllib.request.urlretrieve(json_url, json_filename)
...
 kinetics_id_to_classname = {}
 for k, v in kinetics_classnames.items():
     kinetics_id_to_classname[v] = str(k).replace('"', "")
 ```
 
-#### Define input transform
+#### 입력 형태에 대한 정의
```
Comment (codingbowoo) [inline facebookresearch_pytorchvideo_slowfast.md:81]:
```suggestion
#### 입력 변환에 대한 정의
```
내용상 입력한 비디오 프레임을 텐서로 변환하는 '변환'에 대한 정의로 보여, 수정 제안 드립니다.
다만, `입력 변환` 이라는 표현이 좀 어색한 것 같기도 하네요...

File: facebookresearch_pytorchvideo_slowfast.md [hunk 10]
```diff
@@ -143,54 +143,53 @@ try: urllib.URLopener().retrieve(url_link, video_path)
 except: urllib.request.urlretrieve(url_link, video_path)
 ```
 
-Load the video and transform it to the input format required by the model.
+영상을 불러오고 모델에 필요한 입력 형식으로 변환합니다.
 
 ```python
-# Select the duration of the clip to load by specifying the start and end duration
-# The start_sec should correspond to where the action occurs in the video
+# 시작 및 종료 구간을 지정하여 불러올 클립의 길이 선택합니다.
```
Comment (codingbowoo) [inline facebookresearch_pytorchvideo_slowfast.md:149]:
```suggestion
# 시작 및 종료 구간을 지정하여 불러올 클립의 길이를 선택합니다.
```

File: facebookresearch_pytorchvideo_slowfast.md [hunk 11]
```diff
@@ -143,54 +143,53 @@ try: urllib.URLopener().retrieve(url_link, video_path)
...
+# start_sec는 영상에서 행동이 시작되는 위치와 일치해야합니다.
```
Comment (codingbowoo) [inline facebookresearch_pytorchvideo_slowfast.md:150]:
```suggestion
# start_sec는 영상에서 행동이 시작되는 위치와 일치해야 합니다.
```

File: facebookresearch_pytorchvideo_slowfast.md [hunk 12]
```diff
@@ -143,54 +143,53 @@ try: urllib.URLopener().retrieve(url_link, video_path)
...
-# Initialize an EncodedVideo helper class and load the video
+# EncodedVideo helper 클래스를 초기화하고 영상을 불러옵니다.
 video = EncodedVideo.from_path(video_path)
 
-# Load the desired clip
+# 원하는 클립을 불러옵니다.
 video_data = video.get_clip(start_sec=start_sec, end_sec=end_sec)
 
-# Apply a transform to normalize the video input
+# 영상 입력을 정규화하기 위해 변형(transform) 함수를 적용합니다.
```
Comment (codingbowoo) [inline facebookresearch_pytorchvideo_slowfast.md:160]: 주석에서 의미하는 바는 transform 메소드를 칭하기보다는 변환을 적용한다는 일반적인 뜻인 것으로 보입니다. 
그러나 헷갈릴 수 있으니 @cpprhtn 님께서 처음 적어주신 것처럼 transform 함수라고 명시해 봤는데 어떠실까요?
```suggestion
# 영상 입력을 정규화하기 위한 변환(transform 함수)을 적용합니다.
```

**Review by codingbowoo — CHANGES_REQUESTED** [review]
일부 의미상 번역 수정 및 논의가 필요할 것 같습니다.

File: facebookresearch_pytorchvideo_slowfast.md (hunk 12 above)
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md]: 좋은것 같습니다 :) LGTM 👍

File: facebookresearch_pytorchvideo_slowfast.md (hunk 6 above)
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_slowfast.md:21]: 감사합니다!

**Review by hyoyoung — APPROVED** [review]
good

**Review by codingbowoo — APPROVED** [review]
